src/saveToCase.js

```diff
@@ -568,6 +687,25 @@ define([
                         return;
                     }
                 }
+                
+                var attachmentRegex = /\/case\/attachment\/(\w+)\/@src/,
+                    attachRet = path.match(attachmentRegex);
+                if (attachRet) {
+                    basePath = path.replace(attachmentRegex, "");
+                    mug = form.getMugByPath(basePath);
+                    if (mug && mug.__className === "SaveToCase") {
+                        var attachProperties = mug.p.attachment_property,
+                            nodeName = attachRet[1];
+                        if (!attachProperties) {
+                            attachProperties = {};
```

Comment:
I think you may also want to set `mug.p.attachment_property` here. Otherwise it seems that you'll get a type error below when you try to set its properties. Might want to add a test for that logic path.